lib/led/ledcontrol.js

```diff
@@ -72,6 +72,13 @@ function LedControl(opts) {
   this.sendDigit = controller.sendDigit;
   this.initialize = controller.initialize;
 
+  // Functions from the HT1632C controller. Perhaps we can find a more declarative
+  // way to copy functions from controller to LEDcontrol (mixin?)
```

Comment:
Yes, `LedControl` is a nightmare and I want to toss it for a complete overhaul that follows the Component/Controller model (as seen in just about every other class). Unfortunately, this is a huge undertaking and I don't have the time to do it and no one has stepped forward to tackle it.